src/ScssAssetConverter.php

```diff
@@ -66,18 +66,18 @@ public function convert($asset, $basePath)
         return $cssAsset;
     }
 
-    private function getExtension(string $filename): string
+    protected function getExtension(string $filename): string
```

Comment:
This change seems strange to me, why would anyone want to override this behaviour, since it's so trivial?